template/partial/settings.html

```diff
@@ -0,0 +1,49 @@
+<form class="ui form signup-form">
+	<div class="two fields">
+		<div class="field">
+			<label>Email</label>
+			<input type="text" name="email" id="email" placeholder="Email">
+			<div class="ui basic red right pointing prompt label transition hide"></div>
+		</div>
+		<div class="field">
+			<label>Name</label>
+			<input type="text" name="name" id="name" placeholder="Your complete name">
+			<div class="ui basic red left pointing prompt label transition hide"></div>
+		</div>
+	</div>
+	<div class="two fields">
```

Comment:
@MehaKaushik "two fields" is not a name for this div. It's a class name which is used for style sheets. These class names must be short and simple to increase page load and readability.

Also, these class names are specified in Semantic's stylesheets. We cannot really do anything with such names. Such class names are all over our codebase.

Reference: [Semantic Columns](https://semantic-ui.com/elements/container.html#/examples) | [Semantic Fields](https://semantic-ui.com/collections/form.html)